e2e/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use the official Playwright image as a base
+FROM mcr.microsoft.com/playwright:v1.48.1-noble
+
+# Install make and any other necessary dependencies
+RUN apt-get update && apt-get install -y make
+
+# Set the working directory to the root of the project inside the Docker container
+WORKDIR /app
+
+# Copy the required files and directories from the root of the project into the Docker container
+COPY Makefile /app/Makefile
+COPY app /app/app
```

Comment:
do we need the app in this dockerfile? i thought this dockerfile was just for playwright. is there a way for playwright to be able to access an app running locally outside of the playwright container? if so that'd be ideal so we can use the single playwright container to test multiple different apps without needing to copy all the apps into the playwright container